db/schema.rb

```diff
@@ -239,6 +239,9 @@
     t.uuid "assessmentable_id"
     t.string "assessmentable_type"
     t.string "value_type"
+    t.string "section"
```

Comment:
Might be worth adding a DB level null constraint to this (and probably `value_type` too) in a future PR once all of the data has been backfilled?

Reply:
done in https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/1281